packages/astro-frontend/.env

```diff
@@ -0,0 +1,13 @@
+# Strapi
+STRAPI_API_URL=http://localhost
+STRAPI_API_PORT=1337
+STRAPI_API_TOKEN=f081c9729f6883e8c97567891bc8f2b0dbb02756be4381e3f594ad38122e85d65bf72eec2382e176b2479829d36169c77317059ff3508c083b0511b9c824c18a182b862202cfa146286ae4f812dfd400b772b27000ee3862322a5b50cf26c9d6eaa15fab5305428ac7c361967ab9046f8a4c63cf707f3d6f79df7cc8eece7a33
```

Comment:
minor, but if this key can be anything, could we change it to something like `development_dummy_key` or something?
I already see this flagged as a warning during the security assessment 😅